google-cloud-bigtable/lib/bigtable/table_admin_client.rb

```diff
@@ -0,0 +1,242 @@
+# frozen_string_literal: true
+
+require "google/cloud/bigtable/admin/v2"
+require "google/bigtable/admin/v2/table_pb"
+require "google/bigtable/admin/v2/bigtable_table_admin_pb"
+
+module Bigtable
+  # Protobuf structure references
+  Table = Google::Bigtable::Admin::V2::Table
+  ColumnFamily = Google::Bigtable::Admin::V2::ColumnFamily
+  GcRule = Google::Bigtable::Admin::V2::GcRule
+  ColumnFamilyModification =
+    Google::Bigtable::Admin::V2::ModifyColumnFamiliesRequest::Modification
+
+  class TableAdminError < StandardError; end # :nodoc:
```

Comment:
Where is this error class being used? This should probably inherit from `Google::Cloud::Error`.

Reply:
@blowmage Right now not in use. Removing it.